homeassistant/components/netdata/sensor.py

```diff
@@ -145,6 +147,72 @@ async def async_update(self):
         )
 
 
+class NetdataAlarms(Entity):
+    """Implementation of a Netdata alarm sensor."""
+
+    def __init__(self, netdata, name, host, port):
+        """Initialize the Netdata alarm sensor."""
+        self.netdata = netdata
+        self._state = None
+        self._name = name
+        self._host = host
+        self._port = port
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"{self._name} Alarms"
+
+    @property
+    def state(self):
+        """Return the state of the resources."""
+        return self._state
+
+    @property
+    def device_info(self) -> Optional[Dict[str, Any]]:
+        """Return host information, e.g. the url to netdata dashboard."""
+        return {"url": f"http://{self._host}:{self._port}"}
+
+    @property
+    def entity_picture(self):
+        """Status symbol if type is symbol."""
+        if self._state == "ok":
+            return "https://cdn2.iconfinder.com/data/icons/function_icon_set/circle_green.png"
+        elif self._state == "warning":
+            return "https://cdn2.iconfinder.com/data/icons/function_icon_set/warning_48.png"
+        elif self._state == "critical":
+            return "https://cdn2.iconfinder.com/data/icons/function_icon_set/circle_red.png"
+        else:
+            return "https://cdn2.iconfinder.com/data/icons/function_icon_set/refresh_48.png"
+
+    @property
+    def available(self):
+        """Could the resource be accessed during the last update call."""
+        return self.netdata.available
+
+    async def async_update(self):
+        """Get the latest alarms from Netdata REST API."""
+        import json
+
+        await self.netdata.async_update()
+        info = json.loads(self.netdata.api.alarms)
+        self._state = "unknown"
```

Comment:
Please use `None` to represent unknown state.

Reply:
Changed.